src/modules/RA/DataSource/views/List.js

```diff
@@ -19,8 +19,7 @@ import {
   sourceTypes,
   sourceTypeChoices,
 } from '..';
-import CommonBulkActionButtons
-  from '../../../../components/react-admin/CommonBulkActionButtons';
+import CommonBulkActionButtons from '../../../../components/react-admin/CommonBulkActionButtons';
```

Comment:
Pourquoi ce fichier a changé ?

Reply:
qu'est ce qui a changé?

Reply:
le retour à la ligne après `CommonBulkActionButtons` 😉

Reply:
Le rouge c'est ce qu'il y avait avant, le vert maintenant, c'est principalement du reformatage pour cet exemple. Je pense que tu as sauvegarder le fichier et que ton éditeur l'a reformaté tout seul.

Reply:
> The carrier return after `CommonBulkActionButtons` wink
*carriage return*

Reply:
It has been reverted.